src/components/site/SiteWithoutStyle.tsx

```diff
@@ -9,10 +9,10 @@ export const SiteWithoutStyle = (props: SiteProps) => {
   const { shouldShowCelebration, onCelebrationXClicked, isSideNavOpen, url } = props;
   const { classes, ...propsWithoutClasses } = props;
   return <React.Fragment>
+  {1 < 2 && <CelebrationModal numHeartpointsAwarded={10} onXClicked={onCelebrationXClicked} />}
```

Comment:
what's `1 < 2 ` for?

Reply:
Just a dumb way to keep the modal open without having to click through the Facebook login process every time I rebuilt.

I will change it back to `{shouldShowCelebration && ...` on my next commit. Sorry. I should have removed it before pushing.